### PR TITLE
Created 2 components

### PR DESCRIPTION
**Pull request by christian-withjoy**
* `FloatingPane`, an absolutely positioned wrapper for an arbitrary element,
* `DropDownButton`, takes a button (in reality, this is arbitrary) and an arbitrary element and creates a toggleable dropdown, using the `FloatingPane` element.

~~Still need to add transition animations to the 2nd one.~~
Looks OK without animations for right now, I think.

### CODE REVIEW

**Comment by christian-withjoy** [comment]
@mcorrigan89 added observable to detect clicks

File: src/components/empty/empty.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
```
Comment (mcorrigan89) [inline src/components/empty/empty.tsx]: What's going on with this file?

File: src/components/dropDownButton/DropDownButton.tsx [hunk 2]
```diff
@@ -0,0 +1,94 @@
+import * as React from 'react';
+import { css } from 'glamor';
+import { error } from 'util';
+import { Button } from '../button/Button';
+import { DropDown } from '../dropdown/Dropdown';
+import { FloatingPane } from '../floatingPane/FloatingPane'
+import { VelocityTransitionGroup } from 'velocity-react';
+import { Observable } from 'rxjs';
```
Comment (mcorrigan89) [inline src/components/dropDownButton/DropDownButton.tsx]: you'll want to only import the operators and Observables that you use. so here it should be 
```import { Observable } from 'rxjs/Observable';```

File: src/components/floatingPane/FloatingPane.tsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+export interface Props {
+  element: React.ReactElement<any>
+}
+export class FloatingPane extends React.Component<Props, {}> {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    const outerContainer = css({
```
Comment (mcorrigan89) [inline src/components/floatingPane/FloatingPane.tsx]: These css rules should probably be outside of the render function.

File: src/components/empty/empty.tsx (hunk 1 above)
Comment (christian-withjoy) [reply src/components/empty/empty.tsx]: put it there for copypasting as a starting template, want me to remove ?

Comment (christian-withjoy) [reply src/components/empty/empty.tsx]: @mcorrigan89  probably some way to do it in the editor (I'm 99% sure atom has a way to save templates)

Comment (mcorrigan89) [reply src/components/empty/empty.tsx]: yeah, this would be better as a snippet or something in the IDE, not committed to the codebase

**Comment by christian-withjoy** [comment]
@mcorrigan89 requested changes made

**Pull request merged**